bc/subscription/tasks.py

```diff
@@ -35,9 +36,21 @@ def process_filing_webhook_event(fwe_pk) -> FilingWebhookEvent:
     filing_webhook_event.subscription = subscription
     filing_webhook_event.save()
 
-    message = MASTODON_POST_TEMPLATE.format(
+    if filing_webhook_event.short_description:
+        description = filing_webhook_event.short_description
+    else:
+        docket_length = len(subscription.docket_name)
+        available_space = 500 - len(POST_TEMPLATE) - docket_length
+        if len(filing_webhook_event.long_description) > available_space:
+            description = trunc(
+                filing_webhook_event.long_description, available_space, "...👇"
+            )
+        else:
+            description = filing_webhook_event.long_description
```

Comment:
I think all of this should be in the Mastodon template as a method. It's a lot of detailed stuff inside a task that should really just be focused on sending posts and tweets.

We should also store the max Mastodon length in a variable we import wherever we use it. Who knows if it'll get longer.

Reply:
I agree. I think We can add another property on the `FilingWebhookEvent` model to get the description, something like:

```python 
@property
def description(self) -> str:
        if self.short_description:
            return self.short_description
        return self.long_description
```

and We can also add a method in the Mastodon template to truncate the string if needed